larray_eurostat/tsv.py

```diff
@@ -1,35 +1,143 @@
+###############################################################################################
+## OLD EUROSTAT API was deprecated anno 23 october 2023. 
+## Minor modification were made in below functionality (which supports caching).
+##
+## CHANGES: - change of base url for compatibility with new API
```

Comment:
This kind of "historical"/"changelog" comment is not needed/discouraged when using Git.